src/ui/router/ApplicationRouter.js

```diff
@@ -126,7 +126,21 @@ class ApplicationRouter extends EventEmitter {
     }
 
     /**
-     * Navgate to given hash and update current location object and notify listeners about location change
+     * Navigate to given hash and update current location object and notify listeners about location change
+     *
+     * @param {Array<Object>} objectPath Object path of a given Domain Object
+     *
+     * @returns {Boolean}
+     */
+    isNavigatedObject(objectPath) {
```

Comment:
Ooops, this is duped now because of https://github.com/nasa/openmct/pull/4114. They are identical. Mind removing before we merge? Thanks!

Reply:
done, added JSDoc